### ISSUE TITLE
Safari: Impossible to reach "Previous Page" and "Next Page" buttons using keyboard

### ISSUE THREAD

**Issue by felipebochehin87** [swissspidy/web-stories-wp#11248]
## Bug Description

Impossible to reach "Previous Page" and "Next Page" buttons using Tab. The focus changes from the canvas to the story title, to the main block, etc... but never reaches the "Previous Page" and "Next Page" buttons.

## Expected Behaviour

"Previous Page" and "Next Page" buttons should be reachable when user hits Tab or Shift + Tab.

## Steps to Reproduce

1. Open Story editor in a Safari window
2. Hit Tab button multiple times too
3. Hit Shift + Tab button multiple times too

## Screenshots

https://user-images.githubusercontent.com/96055551/162930219-9360ca96-70d7-4ced-9589-73669540b1b1.mp4

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by swissspidy** [comment]
Can't reproduce this, even on a blank new story:

https://user-images.githubusercontent.com/841956/162920853-8ab76001-ae2b-4047-89e0-b18544f26008.mov

**Comment by felipebochehin87** [comment]
@swissspidy good morning! :) 

Please let me ask you, are you using the main branch or any other specific branch? I tried in a few branches + the main one and I still can't reach the buttons...

**Comment by felipebochehin87** [comment]
BTW, I added a video to the description.

**Comment by swissspidy** [comment]
Tested on `main` & Safari 15.4, as well as Chrome 100.

**Comment by felipebochehin87** [comment]
Weird, me too. Chrome and Firefox look good. Safari 15.4 (17613.1.17.1.13) is not (for me).

**Comment by BrittanyIRL** [comment]
This should be resolved - tested with Safari 15.4 (17613.1.17.1.13)

![Kapture 2022-04-18 at 16 49 12](https://user-images.githubusercontent.com/10720454/163894261-02daa94f-5a7f-4ea3-9963-7df6477b80b2.gif)

@felipebochehin87 are you still seeing this in staging?

**Comment by BrittanyIRL** [comment]
@merapi also stealing this because we're working on canvas focus updates and i think we resolved the focus trapping.

**Comment by felipebochehin87** [comment]
This is so weird, I still see it in Staging. I am using MacBook 2019, 16-inch, macOS Monterey version 12.3.1. Safari 15.4 (17613.1.17.1.13).


@BrittanyIRL

**Comment by BrittanyIRL** [comment]
This still works fine for me. Try QAing it again with #11003 and see if that makes a difference for you? Otherwise, this ticket should go into the backlog to watch for when it can be reproduced. I can't get it to trigger at all.

**Comment by felipebochehin87** [comment]
@BrittanyIRL @BrookeGraham @swissspidy @bmattb I still can reproduce this issue, but apparently I am the only one. Should we kick this back for more digging (_I wonder if it's possible to dig more than we have already dug..._) or should we move this ahead and keep an eye on Safari to check if the bug happens to someone else?

**Comment by swissspidy** [comment]
As per Brittany's previous comment, this should go back into the backlog for now until this can be reliably reproduced by more people. No other action for now. It's not a high priority issue.

**Comment by felipebochehin87** [comment]
Thanks!
